tests/unit/commands/install-test.js

```diff
@@ -156,16 +156,6 @@ describe('install command', function() {
       });
     });
 
-    it('runs the package name blueprint task when given github/name and args', function() {
-      return command.validateAndRun(['ember-cli/ember-cli-qunit']).then(function() {
```

Comment:
we should probably keep the test, but then test for the yet-to-be-implemented warning message